packages/components/src/interfaces/internal/ArticlePageHeader.ts

```diff
@@ -6,12 +6,14 @@ import type { CollectionCardProps } from "./CollectionCard"
 import type { LinkComponentType } from "~/types"
 
 export const ArticlePageHeaderSchema = Type.Object({
-  summary: Type.Array(Type.String(), {
-    title: "Article summary",
-    description:
-      "The summary of the article page's content. Having multiple items will display them as a list.",
-    minItems: 0,
-  }),
+  summary: Type.Optional(
```

Comment:
issue: wanted to check here, I thought the page summary should be a required prop?